openpype/modules/kitsu/kitsu_module.py

```diff
@@ -0,0 +1,477 @@
+"""Kitsu module."""
+
+import click
+import os
+import re
+
+import gazu
```

Comment:
It is really important that this file contains entrypoint of the module describing the public face but not real implementation. Imports here must not contain any dependencies and functions should not do really complex logic. Imagine that you don't have `gazu` python module installed and expect that any side file of the module can crash but this file still must be possible to import.

For example the whole sync function should be in separated file that is imported only when sync is called.